dialog/lib/dialog.ts

```diff
@@ -383,17 +389,17 @@ export class Dialog extends LitElement {
 
   private dialogClosedResolver?: () => void;
 
-  private async performTransition(shouldDispatchAction: boolean) {
+  private async performTransition(/*shouldDispatchAction: boolean*/) {
```

Comment:
@sorvell do you have more context on when we should or should not dispatch the action event on a transition?